### ISSUE TITLE
XML configs duplicated, --configFile option doesn't work

### ISSUE THREAD

**Issue by glennhickey** [ComparativeGenomicsToolkit/cactus#151]
There are two config files in the source, `src/cactus/cactus_config.xml`, and `src/cactus/cactus_progressive_config.xml`.  The seem to be the same, except `cactus_progressive_config.xml` has comments.  They both turn up in the code
```
src/cactus/pipeline/cactus_workflowTest.py:        self.configFile = os.path.join(cactusRootPath(), "cactus_config.xml")
src/cactus/preprocessor/cactus_preprocessor.py:        # Parse the "preprocessor" config xml element
src/cactus/preprocessor/cactus_preprocessor.py:    parser.add_argument("--configFile", default=os.path.join(cactusRootPath(), "cactus_progressive_config.xml"))
src/cactus/preprocessor/cactus_preprocessorTest.py:        configFile = os.path.join(self.tempDir, "config.xml")
src/cactus/progressive/cactus_progressiveTest.py:        configWrapper = ConfigWrapper(ET.parse(os.path.join(cactusRootPath(), "cactus_progressive_config.xml")).getroot())
src/cactus/progressive/projectWrapper.py:                                      "cactus_progressive_config.xml")
src/cactus/progressive/projectWrapper.py:            os.path.join(self.workingDir, "config.xml"))
src/cactus/reference/cactus_referenceTest.py:    config = ET.parse(os.path.join(cactusRootPath(), "cactus_progressive_config.xml")).getroot()
src/cactus/shared/configWrapper.py:""" Interface to the cactus config xml file used
src/cactus/shared/experimentWrapper.py:            config = os.path.join(cactusRootPath(), "cactus_config.xml")
src/cactus/shared/experimentWrapper.py:            config = os.path.join(cactusRootPath(), "cactus_progressive_config.xml")
```

I'm not too sure what's used for what.  What I've learned the hard way so far:  
* If I want to turn off a preprocessor, I need to edit `src/cactus/cactus_config.xml`.  `--configFile` also works here
* If I want to modify `caf.chunkSize`, then I need to edit `src/cactus/cactus_config.xml`.  `--configFile` *will not* override it. 

So I think we need to 
* make sure there is just 1 config xml in cactus
* fix the `--configFile` option to work for all parameters, and disable it until then.

**Comment by glennhickey** [comment]
Apart from the comments, there are a few small differences between the two xml files.  

`cactus_progressive_config` has both `cactus_normalization` and `cactus_check` off by default.  But `checkAssemblyHub` on in `checkHeaders`.  

I'd imagine these are the most up-to-date options on all counts, but am not certain.  

```
xmlstarlet c14n --without-comments src/cactus/cactus_config.xml | xmllint --format - > cactus_config_clean.xml
xmlstarlet c14n --without-comments src/cactus/cactus_progressive_config.xml | xmllint --format - > cactus_progressive_config_clean.xml
diff cactus_config_clean.xml cactus_progressive_config_clean.xml
7c7
<   <preprocessor check="1" checkAssemblyHub="0" memory="littleMemory" preprocessJob="checkUniqueHeaders"/>
---
>   <preprocessor check="1" checkAssemblyHub="1" memory="littleMemory" preprocessJob="checkUniqueHeaders"/>
24c24
<   <normal iterations="2">
---
>   <normal iterations="0">
39c39
<   <check runCheck="1">
---
>   <check runCheck="0">
```

**Comment by glennhickey** [comment]
Fixed in #186.  Can still be better though (#189)
